### DISCUSSION TITLE
integrate speech to text  using javascript , not getting any request from deepgram server side

### DISCUSSION THREAD

**Discussion by vivekwgs** [deepgram/community#624] [General help]
the speech to text concept we using javascript we not any request from deepgram server side ......

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Reply by vivekwgs** [reply]
no we are using deep gram api that was when we speech to it  should transcript get the live transcript speech to text

**Comment by jpvajda** [comment]
@vivekwgs  We'll need more information to try to assist you can you see the message above for the information of what we need to try to help?

**Comment by vivekwgs** [comment]
Yes, I can.

**Comment by vivekwgs** [comment]
we are facing with websocket issues i mea we not any requesting from the
server side in case we are http requests but live streaming part in case we
seem that clear cross check

**Comment by jpvajda** [comment]
@vivekwgs  Thanks but we'll need a bit more like:

*  The` request_id` if you have a question about your requests or transcription responses.
*  The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
*  Any code snippets you can share.

**Comment by vivekwgs** [comment]
Ok, no problem.  we are facing the same problem and that websocket we we
are using in node js and express with database in mongodb we facing with
websocket issue that it seem that
